client/components/domains/register-domain-step/index.jsx

```diff
@@ -506,7 +506,8 @@ class RegisterDomainStep extends React.Component {
 			! Array.isArray( this.state.searchResults ) &&
 			! this.state.loadingResults &&
 			! this.props.showExampleSuggestions;
-		const showFilters = isKrackenUi && ! isRenderingInitialSuggestions && ! this.props.isReskinned;
+		const showFilters =
+			( isKrackenUi && ! isRenderingInitialSuggestions ) || this.props.isReskinned;
```

Comment:
`isKrackenUi` is defined twice (`client/components/domains/register-domain-step/index.jsx` line [501](https://github.com/Automattic/wp-calypso/blob/3acbf529fb061866c8b0eebf7f17c5b93bcb7251/client/components/domains/register-domain-step/index.jsx#L501) & [389](https://github.com/Automattic/wp-calypso/blob/3acbf529fb061866c8b0eebf7f17c5b93bcb7251/client/components/domains/register-domain-step/index.jsx#L389)). 

Why are the definitions different and are the filters affected?

Reply:
> Why are the definitions different and are the filters affected?

Because one of them is only concerning pagination, while the other is concerning filters. I don't think filters are affected by this inconsistency. Although, I agree that the Kraken variable names aren't great.

Reply:
Thanks for elaborating @alshakero. It may become confusing in the future, but for now, it makes sense to let it as is.